pages/vi/vi-github-and-markdown.md

```diff
@@ -66,7 +66,9 @@ After you fork the repository, you will be on your page: &lt;username&gt;/open-l
 
 After you rename it, make sure to post a link to the github.io website you've just created (&lt;username&gt;.github.io) in the [gitter.im chat](https://gitter.im/open-learning-exchange/chat).
 
-![Example of github.io page](images/vi-example-githubio.png)
+> ** @dogi  please look at
+https://YourUserName.github.io and
+ https://rawgit.com/YourUserName/YourUserName.github.io/YourBranchName/#!pages/vi/profiles/YourUserName.md **
```

Comment:
As we [discussed in the other PR](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1343#discussion_r147562721), the personal profile markdown file wouldn't exist at this point if we follow the tutorial from top to down. So the second link might better be removed.

To keep things at minimal, bold texts are probably not necessary. The quotation sort of already indicates it's not normal text. What do you think?

Reply:
Made the changes...Please review. Thanks!!